packages/tempus/test/TestModels/CDR_Model_impl.hpp

```diff
@@ -355,6 +355,7 @@ void CDR_Model<Scalar>::evalModelImpl(
     if (nonnull(W_prec_out)) {
       RCP<Epetra_Operator> M_epetra = Thyra::get_Epetra_Operator(*(W_prec_out->getNonconstRightPrecOp()));
       M_inv = rcp_dynamic_cast<Epetra_CrsMatrix>(M_epetra);
+      M_inv->Print(std::cout);
```

Comment:
I assume these were meant for debugging--just make sure to remove before putting up to Trilinos